swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -99,54 +99,41 @@ Follow the instructions given in this section to develop the service.
-
-    ```ballerina
-    configurable int port = 8290;
-    configurable string hospitalServicesBackend = "http://localhost:9090";
-    configurable string paymentBackend = "http://localhost:9090/healthcare/payments";
-    configurable string host = "smtp.gmail.com";
-    configurable string username = ?;
-    configurable string password = ?;
-    ```
+3. Generate record types corresponding to the payloads from the hospital and payment backend services by providing samples of the expected JSON payloads.
 
-4. Define two [`http:Client`](https://ballerina.io/learn/by-example/#http-client) clients to send requests to the backend services and one [`email:SmtpClient`](https://ballerina.io/learn/by-example/#email-client) client to send emails.
+    The payload from the hospital backend service will be a JSON object similar to the following.
 
-    ```ballerina
-    final http:Client hospitalServicesEP = check initializeHttpClient(hospitalServicesBackend);
-    final http:Client paymentEP = check initializeHttpClient(paymentBackend);
-    final email:SmtpClient smtpClient = check initializeEmailClient();
-
-    function initializeHttpClient(string url) returns http:Client|error => new (url);
-
-    function initializeEmailClient() returns email:SmtpClient|error => new (host, username, password);
+    ```json
+    {
+        "appointmentNumber": 1,
+        "doctor": {
+            "name": "thomas collins",
+            "hospital": "grand oak community hospital",
+            "category": "surgery",
+            "availability": "9.00 a.m - 11.00 a.m",
+            "fee": 7000.0
+        },
+        "patient": {
+            "name": "John Doe",
+            "dob": "1940-03-19",
+            "ssn": "234-23-525",
+            "address": "California",
+            "phone": "8770586755",
+            "email": "johndoe@gmail.com"
+        },
+        "hospital": "grand oak community hospital",
+        "confirmed": false,
+        "appointmentDate": "2023-10-02"
+    }
     ```
 
-    > **Note:** The argument to the `new` expression is the URL for the backend service. 
-    > 
-    > Alternatively, the clients can be initialized directly with `new` expressions, but a separate function is used to aid with testing.
-    > 
-    > ```ballerina
-    > final http:Client hospitalServicesEP = check new (hospitalServicesBackend);
-    > final http:Client paymentEP = check new (paymentBackend);
-    > ```
+    ![Define records](/learn/images/integration-tutorials/sending-emails-from-a-service/define_records.gif)
 
-5. Define records corresponding to the request payload and response payloads.
+    The generated records will be as follows.
 
     ```ballerina
     type Patient record {|
```

Comment:
Have to be open?

Reply:
Yes and might have retake the gifs

Reply:
Yes and might have to retake some of the gifs